rules/windows/privilege_escalation_disable_uac_registry.toml

```
@@ -18,7 +18,65 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Disabling User Account Control via Registry Modification"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Disabling User Account Control via Registry Modification
+
+Windows User Account Control (UAC) allows a program to elevate its privileges (tracked as low to high integrity levels)
+to perform a task under administrator-level permissions, possibly by prompting the user for confirmation.
```

The last part of this sentence "possibly by prompting the user for confirmation" seems out of place. Is it needed in this description of UAC?

---

Yeah, UAC is the mechanism that is responsible for prompting the user for confirmation when needed

---

Maybe it can be it's own sentence, for example:

```suggestion
to perform a task under administrator-level permissions. Windows UAC usually prompts the user to confirm the program's privilege elevation.
```